Dragon.h

```diff
@@ -1,11 +1,24 @@
 #pragma once
 
+#include <string>
 #include "Character.h"
 
 struct Dragon : Character
-{
-    void attack(Character& other) override;
-private:
-    const std::string name;
-};
+{   
+    Dragon
+    (
+        std::string name_, 
+        int hitPoints_, 
+        int armor_
+    );
```

Comment:
Do not put `(` on its own line.  you are confusing them with `{ }`
- [ ] revise

Reply:
ok done